metadata.yaml

```diff
@@ -1,6 +1,7 @@
-# Copyright 2021 Chris
+# Copyright 2021 Canonical Ltd.
```

Comment:
I miss several entries in this file: docs, issues, maintainers...

Reply:
Indeed - those can come in later changes though.